make dockers_dev failing

**BUG REPORT**

1. What were you trying to achieve?
Make the development docker images

2. What are the expected results?
Successfully create the development docker images

3. What are the received results?
Step 3/4 : COPY mainflux-$SVC /exe
COPY failed: stat /var/lib/docker/tmp/docker-builder572195106/mainflux-dev_users: no such file or directory
Makefile:82: recipe for target 'docker_dev_users' failed
make: *** [docker_dev_users] Error 1

4. What are the steps to reproduce the issue?
make && make dockers_dev

5. In what environment did you encounter the issue?
Pop!_OS 18.04

6. Additional information you deem important:
It looks like this happened in #929.  I will revert some of the changes to find what caused the break.

--- a/Makefile
+++ b/Makefile
@@ -26,7 +26,7 @@ define make_docker
 endef
 
 define make_docker_dev
-	$(eval svc=$(subst docker_,,$(1)))
+	$(eval svc=$(subst docker_dev_,,$(1)))
 
 	docker build \
 		--no-cache \

